presidio-image-redactor/presidio_image_redactor/bbox.py

```diff
@@ -0,0 +1,135 @@
+from presidio_image_redactor.entities import ImageRecognizerResult
+from typing import List, Tuple, Dict, Union
+
+
+class BboxProcessor:
+    """Common module for general bounding box operators."""
+
+    def get_bboxes_from_ocr_results(
+        self,
+        ocr_results: Dict[str, List[Union[int, str]]],
+    ) -> List[Dict[str, Union[int, float, str]]]:
+        """Get bounding boxes on padded image for all detected words from ocr_results.
+
+        :param ocr_results: Raw results from OCR.
+        :return: Bounding box information per word.
+        """
+        bboxes = []
+        item_count = 0
+        for i in range(len(ocr_results["text"])):
+            detected_text = ocr_results["text"][i]
+            if detected_text:
+                bbox = {
+                    "left": ocr_results["left"][i],
+                    "top": ocr_results["top"][i],
+                    "width": ocr_results["width"][i],
+                    "height": ocr_results["height"][i],
+                    "conf": float(ocr_results["conf"][i]),
+                    "label": detected_text,
+                }
+                bboxes.append(bbox)
+                item_count += 1
+
+        return bboxes
+
+    def get_bboxes_from_analyzer_results(
+        self,
+        analyzer_results: List[ImageRecognizerResult],
+    ) -> List[Dict[str, Union[str, float, int]]]:
+        """Organize bounding box info from analyzer results.
+
+        :param analyzer_results: Results from using ImageAnalyzerEngine.
+
+        :return: Bounding box info organized.
+        """
+        bboxes = []
+        for i in range(len(analyzer_results)):
+            result = analyzer_results[i].to_dict()
+
+            bbox_item = {
+                "entity_type": result["entity_type"],
+                "score": result["score"],
+                "left": result["left"],
+                "top": result["top"],
+                "width": result["width"],
+                "height": result["height"],
```

Comment:
Along with the suggested Bbox class change, this can be a one liner with spread operator

Reply:
Leaving as-is and marking as resolved (see above response from me about the `Bbox` class).